src/fparser/common/tests/test_readfortran.py

```diff
@@ -340,6 +340,258 @@ def test_base_free_continuation(log):
     assert result == expected
 
 
+def check_include_works(fortran_filename, fortran_code, include_info,
+                        expected, tmpdir, ignore_comments=True):
+    '''Utility function used by a number of tests to check that a include
+    files work as expected.
+
+    '''
+    try:
+        oldpwd = tmpdir.chdir()
+        cwd = str(tmpdir)
+
+        # Create the program
+        with open(os.path.join(cwd, fortran_filename), "w") as cfile:
+            cfile.write(fortran_code)
+        for include_filename in include_info.keys():
+            with open(os.path.join(cwd, include_filename), "w") as cfile:
+                cfile.write(include_info[include_filename])
+        from fparser.common.readfortran import FortranFileReader
```

Comment:
pylint complains that FortranFileReader has already been imported.

Reply:
And it is quite right. Delete line. Sorry. I thought I'd picked up all pylint errors.